portafly/src/components/developer-accounts/modals/BaseModal.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useMemo } from 'react'
```

Comment:
Do you think all modals (or at least BaseModal) can be a separate PR?

Reply:
I think the idea is to create and merge smaller PRs, having this one as a reference only so I wouldn't be worried about it.